releaseNotes/AzureSpringCloudFeatureManagement.md

```diff
@@ -27,15 +27,15 @@
 * The Feature Management libraries have have renamed to match the structure and naming convention in the azure-sdk-for-java.
 * Targeting, a new built in Feature Filter type.
   * Support for rolling out features to a target audience has been added through built in feature filters.
-  * Targeting enables developers to progressively roll out features to a target audience that can be increased gradually.  For more information on the concept of targeting and how to use this new feature take a look at the project's [readme](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-spring-cloud-feature-management#targetingfilter).
+  * Targeting enables developers to progressively roll out features to a target audience that can be increased gradually.  For more information on the concept of targeting and how to use this new feature take a look at the project's [readme](https://github.com/Azure/azure-sdk-for-java/tree/azure-spring-cloud-feature-management-web_2.10.0/sdk/appconfiguration/azure-spring-cloud-feature-management#targetingfilter).
 * TimeWindowFilter now supports ISO 8601 Dates.
 
 <!-- LINKS -->
-[docs]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-spring-cloud-feature-management
+[docs]: https://github.com/Azure/azure-sdk-for-java/tree/azure-spring-cloud-feature-management-web_2.10.0/sdk/appconfiguration/azure-spring-cloud-feature-management
 [package]: https://mvnrepository.com/artifact/com.azure.spring/azure-spring-cloud-feature-management
 [samples]: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/appconfiguration/feature-management-sample
-[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-spring-cloud-feature-management
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/azure-spring-cloud-feature-management-web_2.10.0/sdk/appconfiguration/azure-spring-cloud-feature-management
```

Comment:
So it's correct that this is 2.10 and the other is 2.11?

Reply:
Yes, the feature management library didn't have a release every time the provider library did.